tests/js/session-tests.js

```diff
@@ -172,7 +172,7 @@ module.exports = {
                 TestCase.assertEqual(session.config.url, config.sync.url);
                 TestCase.assertEqual(session.config.user.identity, config.sync.user.identity);
                 TestCase.assertEqual(session.state, 'active');
-                user.logOut();
+                return user.logOut();
```

Comment:
Would it be too big of an unrelated change to make this function an `async function` and awaiting the `user` and then awaiting the `user.logOut()` afterwards? That will reduce the nesting caused by `.then` and the code calling this won't get the result of `logOut` returned (which it shouldn't need).

Reply:
It will require some work since we are spawning a new process (at least, it's more than a 10 minutes hack :smile:)

Reply:
What do you mean by "since we are spawning a new process"?